lib/trainer/test_parser.rb

```diff
@@ -120,6 +132,97 @@ def test_group_and_name(testable_summary, test, xcpretty_naming)
       return group, name
     end
 
+    def parse_xcresult(path)
+      # Executes xcresulttool to get JSON format of the result bundle object
+      result_bundle_object_raw = `xcrun xcresulttool get --format json --path #{path}`
+      result_bundle_object = JSON.parse(result_bundle_object_raw)
```

Comment:
if `xcrun` fails, I think it returns back "" (or maybe stderr?) which will cause a pretty cryptic JSON.parse exception.
Let's test `$?.success?` and raise something more friendly.